src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react"
+import axios from "axios"
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([])
+  const apiUrl = "https://backend-project-d9id.onrender.com/api/articles"
+
+  useEffect(() => {
+    const fetchArticles = async () => {
+      try {
+        const response = await axios.get(apiUrl)
+        setArticles(response.data.articles)
+      } catch (error) {
+        console.error("Error fetching articles:", error)
+      }
+    }
+
+    fetchArticles()
+  }, [apiUrl])
+
```

Comment:
There is currently no indication on your app that let's the user know that the page is loading when we are waiting on the data from the api, could you add something in here that will handle this?